src/Microsoft.Identity.Web.Diagnostics/IdHelper.cs

```diff
@@ -34,9 +36,14 @@ internal static class IdHelper
                return version[1];
            });
 
+        public static string GetIdWebVersion()
+        {
+            return s_idWebVersion.Value;
+        }
+
         public static string CreateTelemetryInfo()
         {
-            return string.Format(CultureInfo.InvariantCulture, Constants.IDWebSku + s_idWebVersion.Value);
+            return string.Format(CultureInfo.InvariantCulture, IDWebSku + GetIdWebVersion());
```

Comment:
Could you please explain why this implementation is better (apart from the migration of the constant to this file)?

Reply:
reverted